client/executor/common/src/error.rs

```diff
@@ -25,92 +25,95 @@ use wasmi;
 pub type Result<T> = std::result::Result<T, Error>;
 
 /// Error type.
-#[derive(Debug, derive_more::Display, derive_more::From)]
+#[derive(Debug, thiserror::Error)]
+#[allow(missing_docs)]
 pub enum Error {
 	/// Unserializable Data
```

Comment:
remove doc comments here for unification?

Reply:
I would like to wrap this up and then do another sweep in a separate PR.